kolibri/core/assets/src/views/StorageNotification.vue

```diff
@@ -114,20 +123,18 @@
         return message;
       },
       closeBanner() {
-        // TODO: Store preference after closure
-        this.bannerOpened = false;
+        localStorage.setItem('download_removed', this.lastDownloadRemoved);
```

Comment:
Not suggesting a change here, just a point of interest for you. Another way of handling this would be to use the getter setter pattern for the showBanner computed prop. More info here (but no need to change this!): https://v2.vuejs.org/v2/guide/computed.html#Computed-Setter